Tasks/AzureAppConfigurationPushV10/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure AppConfiguration Push
+
+### Overview
+
+This task is used to push key-values from a configuration file to your  [App Configuration store](https://learn.microsoft.com/azure/azure-app-configuration/quickstart-azure-app-configuration-create). 
+
+## Contact Information
+
+Please report a problem to [AzureAppConfig@microsoft.com](AzureAppConfig@microsoft.com) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.
```

Comment:
I think you can have something like this:
`Please report a problem to <AzureAppConfig@microsoft.com>` since the link label is the email url.

Reply:
This has been updated thanks